sample/src/debug/java/com/polidea/cockpit/sample/sampleparams/SamplePresenter.kt

```diff
@@ -28,6 +28,11 @@ class SamplePresenter(override val sampleView: SampleContract.View, override val
         setOnChangeListeners()
         setCallbacks()
         setOnValueSelectedListener()
+        setReadOnlyProperties()
+    }
+
+    private fun setReadOnlyProperties() {
+        Cockpit.setAppVersion(BuildConfig.VERSION_NAME)
```

Comment:
There is missing import for `BuildConfig`:
`import com.polidea.cockpit.sample.BuildConfig`
App doesn't compile at the moment